CONTRIBUTING.md

```diff
@@ -93,6 +93,56 @@ The Continuous Integration (CI) runs automatically when a pull request is opened
 - Compilation: the code will be compiled using [Github Actions](https://github.com/features/actions) under Linux, Windows and MacOS, and the unit tests will be run;
 - SonarCloud will report the code smells, duplications and the code coverage. You have to fix all the relevant code smells and add unit tests to reach the required barrier.
 
+## Committer rights on one or several repositories
+
+### Committer rights
+
+A Committer is a developer who has the right to merge on protected branches on a given repository.
```

Comment:
"protected branches": shouldn't we be more precise, that is, main, integration and release branches?

Reply:
Changed into "main and other protected branches"